setup.py

```diff
@@ -20,13 +20,25 @@
         'Pillow',
         'numpy',
         'scipy',
-        "cffi>=1.4.0",
+        'cffi>=1.4.0',
+        'pytest-runner',
```

Comment:
Why do we now have a run-time dependency on `pytest`? This seems like a _bad_ thing.